aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SqlKeyword.java

```diff
@@ -203,6 +223,21 @@ public static SqlKeyword fromQueryPredicateGroup(@NonNull QueryPredicateGroup.Ty
         return sqlKeyword;
     }
 
+    /**
+     * Retrieve the SQL specific keyword for the query sort order type.
+     * @param sortOrder the query sort order type
+     * @return the SQL specific keyword
+     */
```

Comment:
Since the return value of this method will not be null, you could annotate this method with @NonNull.